HW1/P2/P2b.py

```diff
@@ -1,3 +1,30 @@
+# Your code here
+
 from P2 import *
 
 # Your code here
+
+import findspark
+findspark.init('/home/zelong/spark')
+import pyspark
+import random
+
+sc = pyspark.SparkContext()
+i = range(2000)
+j = range(2000)
+
+random.shuffle(i)
+random.shuffle(j)
+
+rdd_i = sc.parallelize(i, 10)
+rdd_j = sc.parallelize(j, 10)
+
```

Comment:
For randomization, i think it is also possible to randomize the partitions you create on the original data instead of randomizing the points prior to parallelizing the data. The method .repartition() does this.